tests/fixtures/fake-index.json

```diff
@@ -29,6 +29,24 @@
       "Six"
     ]
   },
+  "gnureadline": {
+    "6.3.3": []
+  },
+  "ipython": {
+    "2.1.0": {
```

Comment:
I see you changed the data type under the version key to a dict here, which makes sense. To not further complicate the logic in the FakeRepository helper, let's just use this convention everywhere in this JSON file, so make all the other lists dicts (with only a `''` key), too.